news/_posts/core-weekly/2020-03-09-coreweekly-week-10-2020.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 10 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-03-09
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 2nd to Sunday 8th of March 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear Developers,
+
+As announced last week, PrestaShop 1.7.6.4 is available. This maintenance and security release fixes 9 issues reported on versions 1.7.6.0 to 1.7.6.3 and fixes a critical security issue. Please [read the release note](https://build.prestashop.com/news/prestashop-1-7-6-4-maintenance-release/) for more informations.
+
+You might have noticed that there is a lot of "waiting for QA" Pull Requests awaiting on GitHub. This is because QA team had to test the 1.7.6.4 build last week and is still battle-testing the 1.7.7.0 Alpha build. This huge work requires almost all available QA specialists. The test campaign should be finished this week which will then allow the flow of Pull Requests to go back to normal.
```

Comment:
good news, thanks for this paragraph. community will appreciate the transparency.